airbyte-integrations/connectors/source-stripe/source_stripe/schemas/early_fraud_warnings_with_id.json

```diff
@@ -0,0 +1,55 @@
+{
+	"definitions": {},
+	"$schema": "http://json-schema.org/draft-07/schema#", 
+	"$id": "https://example.com/object1676045371.json", 
+	"title": "Root", 
+	"type": "object",
+	"properties": {
+		"id": {
+			"$id": "#root/id", 
+			"title": "Id", 
+			"type":  ["null", "string"],
+			"default": "",
+			"pattern": "^.*$"
+		},
+		"object": {
+			"$id": "#root/object", 
+			"title": "Object", 
+			"type":  ["null", "string"],
+			"default": "",
+			"pattern": "^.*$"
+		},
+		"actionable": {
+			"$id": "#root/actionable", 
+			"title": "Actionable", 
+			"type":  ["null", "string", "boolean"],
```

Comment:
It looks like this type declaration is strange. So what is the exact type of this field (string or boolean)?